Worktez/worktez
# Dev/organization

### Functionality:
Migrated organization feature

### Solution:
Separated create new organization and create new team and made multiple organizations and teams available

### Risk level:

### How to test:
1. ng serve
2. create new organizations and create new teams
3. check the My-Organizations collection in users collection in db
